schemas/task-reclaim-response.yml

```diff
@@ -0,0 +1,41 @@
+$schema:          http://json-schema.org/draft-04/schema#
+title:            "Task Claim Response"
+description: |
+  Response to a successful task claim
+type:             object
+properties:
+  status: {$ref: "http://schemas.taskcluster.net/queue/v1/task-status.json#"}
+  runId:
+    description: |
+      `run-id` assigned to this run of the task
+    type:         integer
+    minimum:      {$const: min-run-id}
+    maximum:      {$const: max-run-id}
+  workerGroup:
+    description: |
+      Identifier for the worker-group within which this run started.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  workerId:
+    description: |
+      Identifier for the worker executing this run.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  takenUntil:
+    description: |
+      Time at which the run expires and is resolved as `failed`,
+      if the run isn't reclaimed.
+    type:         string
+    format:       date-time
+  credentials:  {$const: task-credentials}
+additionalProperties: false
+required:
+  - status
+  - runId
+  - workerGroup
+  - workerId
+  - takenUntil
```

Comment:
so "credentials" aren't required, but don't we always return them no matter what? I mean, it's not as if the caller has the option of not getting temp creds sent back is there?